src/handlers/handleAudio.ts

```diff
@@ -12,6 +12,11 @@ import urlToText from '@/helpers/urlToText'
 export default async function handleAudio(ctx: Context) {
   try {
     const message = ctx.msg
+
+    if (message.chat.type == 'group' || message.chat.type == 'supergroup') {
```

Comment:
This shouldn't be the default behaviour, instead we should add a flag on the chat like `transcribeAllAudio` which is `true` by default

Reply:
A bit confused here. So, do we have to add a new command to turn off and on the `transcribeAllAudio` flag and store it in DB?